### PR TITLE
added overview chart initialisation

### PR DESCRIPTION
**Pull request by frenchbread**
@apinf/developers Here is the hot fix regarding overview chart. Chart initialisation function was somehow removed from develop branch after merging. Here is the fix, tested it and generally it works but area selection does not force chart refresh now, still zooming works.

### CODE REVIEW

**Comment by frenchbread** [comment]
Tested this one again. And it turns out that the code refactoring that we did (dividing code for rendering charts to separate functions like `drawChart`, `parseData`, `renderCharts`) somehow does not let the lineChart to refresh when some filtering or selection has been made on a `overview` chart.

@brylie Any thoughts at his point? I think we can leave it like this or return to one-function view to have this functionality present.

**Comment by frenchbread** [comment]
@apinf/developers This PR is ready to be merged.

File: client/views/dashboard/charts/charts.js [hunk 1]
```diff
@@ -136,19 +136,36 @@ Template.chartsLayout.created = function () {
 
     var chart = dc.lineChart("#line-chart");
     var countryChart = dc.barChart("#bar-chart");
+    var overview = dc.barChart("#overview-chart");
 
     chart
       .width(1140)
```
Comment (brylie) [inline client/views/dashboard/charts/charts.js]: What happens if you omit the chart width? Does the chart become responsive?

File: client/views/dashboard/charts/charts.js [hunk 2]
```diff
@@ -136,19 +136,36 @@ Template.chartsLayout.created = function () {
...
+      .mouseZoomable(true)
+      .rangeChart(overview)
       .renderArea(true)
       .dotRadius(3)
+      .brushOn(false)
       .renderHorizontalGridLines(true)
       .renderVerticalGridLines(true);
 
+    overview
+      .width(1140)
```
Comment (brylie) [inline client/views/dashboard/charts/charts.js]: Again, try removing the chart width, to see if it improves the flexibility of the chart width.

**Comment by frenchbread** [comment]
@brylie When I remove `width`, this happens:
chart
so it does not respond to parent container width.

File: client/views/dashboard/charts/charts.js (hunk 1 above)
Comment (frenchbread) [inline client/views/dashboard/charts/charts.js]: @brylie Fixed this one. Chart's width is dynamic and fits the parent container on load. Still not responsive though.

File: client/views/dashboard/charts/charts.js (hunk 2 above)
Comment (frenchbread) [inline client/views/dashboard/charts/charts.js]: Done

**Pull request merged**